### ISSUE TITLE
log file has wrong name

### ISSUE THREAD

**Issue by korikuzma** [cancervariants/metakb#369]
### Describe the bug

Log file being created has the wrong name

### Steps to reproduce

1. Run `python3 -m metakb.cli update -r` inside virtual env

2. In a separate terminal (at the root dir)
```shell
$ ls | grep .log
gene.log
```

### Expected behavior

Log file should be named `metakb.log`

### Current behavior

Log file is named `gene.log`

### Possible reason(s)

Gene Normalizer logging config is overriding MetaKB's logging config. We have to configure MetaKB logging before importing any modules that define their own logging config.

### Suggested fix

Having this is the top of the CLI module creates a log file with the expected name:

```python
from metakb.log_handle import configure_logs
configure_logs()
```

Not really the best solution because it causes several ruff errors: `Module level import not at top of file`. 

### Branch, commit, and/or version

branch: staging
commit: [7301382](https://github.com/cancervariants/metakb/commit/730138241a9c48e15cf1cc403b19ba484ea7df2c)

### Additional details

I think that @jsstevenson was working on resolving logging configs in our apps, so this may not be an issue once new releases are made.

**Comment by jsstevenson** [comment]
> I think that @jsstevenson was working on resolving logging configs in our apps, so this may not be an issue once new releases are made.

Yeah, this is fixed in a development branch of the gene normalizer but unfortunately not on the main branch. I can/should probably just make a quick fix.

**Comment by jsstevenson** [comment]
should be fixed by https://github.com/cancervariants/gene-normalization/pull/351

**Comment by korikuzma** [comment]
> should be fixed by [cancervariants/gene-normalization#351](https://github.com/cancervariants/gene-normalization/pull/351)

Confirmed. Closing.
